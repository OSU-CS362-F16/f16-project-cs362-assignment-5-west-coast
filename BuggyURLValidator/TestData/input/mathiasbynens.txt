http://foo.com/blah_blah
http://foo.com/blah_blah/
http://foo.com/blah_blah_(wikipedia)
http://foo.com/blah_blah_(wikipedia)_(again)
http://www.example.com/wpstyle/?p=364
https://www.example.com/foo/?bar=baz&inga=42&quux
http://?df.ws/123
http://142.42.1.1/
http://142.42.1.1:8080/
http://?.ws/?
http://?.ws
http://?.ws/
http://foo.com/blah_(wikipedia)#cite-1
http://foo.com/blah_(wikipedia)_blah#cite-1
http://foo.com/unicode_(?)_in_parens
http://foo.com/(something)?after=parens
http://?.damowmow.com/
http://code.google.com/events/#&product=browser
http://j.mp
ftp://foo.bar/baz
http://foo.bar/?q=Test%20URL-encoded%20stuff
http://????.??????
http://??.??
http://??????.???????
http://-.~_!$&'()*+;=:%40:80%2f::::::@example.com
http://1337.net
http://a.b-c.de
http://a.b--c.de/
http://223.255.255.254
http://0.0.0.0
http://10.1.1.0
http://10.1.1.255
http://224.1.1.1
http://1.1.1.1.1
http://123.123.123
http://10.1.1.1
http://10.1.1.254

