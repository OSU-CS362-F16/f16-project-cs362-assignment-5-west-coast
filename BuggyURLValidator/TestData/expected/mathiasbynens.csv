testcase,scheme,ALLOW_LOCAL_URLS,NO_FRAGMENTS,ALLOW_2_SLASHES,ALLOW_ALL_SCHEMES,regex,http://foo.com/blah_blah,http://foo.com/blah_blah/,http://foo.com/blah_blah_(wikipedia),http://foo.com/blah_blah_(wikipedia)_(again),http://www.example.com/wpstyle/?p=364,https://www.example.com/foo/?bar=baz&inga=42&quux,http://?df.ws/123,http://userid:password@example.com:8080,http://userid:password@example.com:8080/,http://userid@example.com,http://userid@example.com/,http://userid@example.com:8080,http://userid@example.com:8080/,http://userid:password@example.com,http://userid:password@example.com/,http://142.42.1.1/,http://142.42.1.1:8080/,http://?.ws/?,http://?.ws,http://?.ws/,http://foo.com/blah_(wikipedia)#cite-1,http://foo.com/blah_(wikipedia)_blah#cite-1,http://foo.com/unicode_(?)_in_parens,http://foo.com/(something)?after=parens,http://?.damowmow.com/,http://code.google.com/events/#&product=browser,http://j.mp,ftp://foo.bar/baz,http://foo.bar/?q=Test%20URL-encoded%20stuff,http://????.??????,http://??.??,http://??????.???????,http://-.~_!$&'()*+,;=:%40:80%2f::::::@example.com,http://1337.net,http://a.b-c.de,http://223.255.255.254,These URLs should fail (0Â ? correct),http://,http://.,http://..,http://../,http://?,http://??,http://??/,http://#,http://##,http://##/,http://foo.bar?q=Spaces should be encoded,//,//a,///a,///,http:///a,foo.com,rdar://1234,h://test,http:// shouldfail.com,:// should fail,http://foo.bar/foo(bar)baz quux,ftps://foo.bar/,http://-error-.invalid/,http://a.b--c.de/,http://-a.b.co,http://a.b-.co,http://0.0.0.0,http://10.1.1.0,http://10.1.1.255,http://224.1.1.1,http://1.1.1.1.1,http://123.123.123,http://3628126748,http://.www.foo.bar/,http://www.foo.bar./,http://.www.foo.bar./,http://10.1.1.1,http://10.1.1.254
0,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
1,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
2,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
3,{[http][https][ftp]},OFF,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
4,{[http][https][ftp]},OFF,OFF,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
5,{[http][https][ftp]},OFF,OFF,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
6,{[http][https][ftp]},OFF,ON,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
7,{[http][https][ftp]},OFF,ON,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
8,{[http][https][ftp]},OFF,ON,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
9,{[http][https][ftp]},OFF,ON,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
10,{[http][https][ftp]},ON,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
11,{[http][https][ftp]},ON,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
12,{[http][https][ftp]},ON,OFF,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
13,{[http][https][ftp]},ON,OFF,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
14,{[http][https][ftp]},ON,ON,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
15,{[http][https][ftp]},ON,ON,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
16,{[http][https][ftp]},ON,ON,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
17,{[http][https][ftp]},ON,ON,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
18,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
19,{[http][https]},OFF,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
20,{[http][https]},OFF,OFF,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
21,{[http][https]},OFF,OFF,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
22,{[http][https]},OFF,ON,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
23,{[http][https]},OFF,ON,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
24,{[http][https]},OFF,ON,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
25,{[http][https]},OFF,ON,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
26,{[http][https]},ON,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
27,{[http][https]},ON,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
28,{[http][https]},ON,OFF,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
29,{[http][https]},ON,OFF,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
30,{[http][https]},ON,ON,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
31,{[http][https]},ON,ON,OFF,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
32,{[http][https]},ON,ON,ON,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
33,{[http][https]},ON,ON,ON,ON,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
34,{[http][https][ftp]},OFF,OFF,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
35,{[http][https][ftp]},OFF,OFF,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
36,{[http][https][ftp]},OFF,OFF,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
37,{[http][https][ftp]},OFF,OFF,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
38,{[http][https][ftp]},OFF,ON,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
39,{[http][https][ftp]},OFF,ON,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
40,{[http][https][ftp]},OFF,ON,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
41,{[http][https][ftp]},OFF,ON,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
42,{[http][https][ftp]},ON,OFF,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
43,{[http][https][ftp]},ON,OFF,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
44,{[http][https][ftp]},ON,OFF,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
45,{[http][https][ftp]},ON,OFF,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
46,{[http][https][ftp]},ON,ON,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
47,{[http][https][ftp]},ON,ON,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
48,{[http][https][ftp]},ON,ON,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
49,{[http][https][ftp]},ON,ON,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
50,{[http][https]},OFF,OFF,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
51,{[http][https]},OFF,OFF,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
52,{[http][https]},OFF,OFF,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
53,{[http][https]},OFF,OFF,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
54,{[http][https]},OFF,ON,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
55,{[http][https]},OFF,ON,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
56,{[http][https]},OFF,ON,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
57,{[http][https]},OFF,ON,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
58,{[http][https]},ON,OFF,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
59,{[http][https]},ON,OFF,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
60,{[http][https]},ON,OFF,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
61,{[http][https]},ON,OFF,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
62,{[http][https]},ON,ON,OFF,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
63,{[http][https]},ON,ON,OFF,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
64,{[http][https]},ON,ON,ON,OFF,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
65,{[http][https]},ON,ON,ON,ON,[a-z],true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
66,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,true,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,true,true,true,false,false,false,false,false,false,true,true
